### ISSUE TITLE
suggestions for tuning farmland color

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#1691]
Current farmland color is fairly dark and strong, same brightness as the urban land covers, although mapping is frequently on very large areas.  This causes problems with road colors as well as similarity with several other landcovers.  This matter was already discussed in #1006.  In general it would make more sense to have farmyard in the same brightness as the urban landcovers and make farmland brighter.

Here are three suggestions for changing it to a lighter tone - with comparison to other landcover colors it might conflict with:

The current color:

![current](http://www.imagico.de/files/carto-farmland.png)

Suggestion 1:

![current](http://www.imagico.de/files/carto-farmland1.png)

Suggestion 2:

![current](http://www.imagico.de/files/carto-farmland2.png)

Suggestion 3:

![current](http://www.imagico.de/files/carto-farmland3.png)

All improve contrast with urban landuses and mud but reduce contrast with bare_ground and sand.  In reality farmland edging to sand or other bare ground is rare so this might not be that much of a problem but ultimately this of course needs some practical testing

**Comment by kocio-pl** [comment]
I like 1) scheme - maybe this will be enough. Could you make some comparative renderings?

**Comment by imagico** [comment]
I don't plan to do more elaborate testing at the moment - just wanted to put up these ideas since discussion w.r.t. road colors brought up the possibility.  Anyone who wants to implement a color change and make more thorough testing on real world samples is welcome.

Note in #338 it was rightfully mentioned that brown color is often not really that intuitive for farmland - a more greenish tone might therefore be better.  Since grass is now somewhat less yellow there is a bit of room in the bright yellow-green area that could be used.

**Comment by kocio-pl** [comment]
BTW: I think farmyard can be the same color as landuse=residential, since they are basically telling the same (close environment where people live enclosing basic amenities), for me farmyard is just a special kind of residential area.

**Comment by imagico** [comment]
No, farmyard is more like a subtype of industrial, even if there are often residential buildings present.  It makes sense to have a separate color for that.  Sadly a lot of remote mapping these days does not make this distinction and a residential landuse polygon is drawn around any group of buildings in rural areas.

**Comment by dieterdreist** [comment]
sent from a phone

> Am 27.07.2015 um 19:20 schrieb kocio-pl <notifications@github.com>:
> 
> for me farmyard is just a special kind of residential area


TL;DR;
They are more producing than residential places 

***
Inside settlements farmyards are similar to other producing small businesses (workspace and storage, maybe a yard but generally less outdoor space and more compact, dense use), while in the countryside they tend to be bigger and comprise more outdoor areas for animals, more extensive storage (machinery, forage, hay and grass, dung...). The actual implementation also depends on scale (and age: historic one vs modern ones) and producing philosophy and typology (crop vs. cattle/livestock), some might even be better mapped as industrial (combined multi floor breeding, slaughtering and packaging/shipping plants come to mind) https://en.wikipedia.org/wiki/Vertical_farming
https://web.archive.org/web/20050430210118/http://www.agro.nl/innovatienetwerk/doc/agroproductie_parken_n.pdf

**Comment by kocio-pl** [comment]
For me it's not a contradiction - in general they are both living and producing space. Unfortunately this means we shouldn't get rid of this color.

**Comment by dieterdreist** [comment]
sent from a phone

> Am 28.07.2015 um 11:49 schrieb kocio-pl <notifications@github.com>:
> 
> For me it's not a contradiction - in general they are both living and producing space.


as they are both they are significantly different from a pure living space. Just think about noise and smell to get the difference ;-)

**Comment by pnorman** [comment]
Finding a distinct enough color might be hard. I had a tough time when bringing them to the same lightness.

3 is probably too green.


I would suggest waiting till after any changes to green colours.

**Comment by kocio-pl** [comment]
It was suggested lately [to change farmland color to less bright one](https://lists.openstreetmap.org/pipermail/dev/2015-December/028955.html). I'd like to render some mockups to illustrate this idea, but how should one calculate the corresponding @farmland-line color?
